pkg/assembler/backends/keyvalue/certifyVuln.go

```diff
@@ -61,6 +61,55 @@ func (n *certifyVulnerabilityLink) Key() string {
 	}, ":"))
 }
 
+// Helper function to remove vulnerability links. This works by setting all the links expect the specified linkID.
+func (c *demoClient) removeLinks(ctx context.Context, linkID string, links []string, col string, id string) error {
+	var newLinks []string
+	for _, id := range links {
+		if id != linkID {
+			newLinks = append(newLinks, id)
+		}
+	}
+
+	// Update the entity in the KeyValue store
+	return c.kv.Set(ctx, col, id, newLinks)
```

Comment:
This isn't correct. It is replacing the vuln and package structs/nodes with a string array. You need to get / modify / and set those structs. Should probably use the `setkv()` helper (https://github.com/guacsec/guac/blob/main/pkg/assembler/backends/keyvalue/backend.go#L248)

Reply:
This is still not right and doing the same thing.